.ci/Jenkinsfile_baseline_capture

```diff
@@ -35,7 +35,6 @@ kibanaPipeline(timeoutMinutes: 210) {
               kibanaPipeline.withCiTaskQueue([parallel: 2]) {
                 catchErrors {
                   tasks([
-                    kibanaPipeline.functionalTestProcess('oss-baseline', './test/scripts/jenkins_baseline.sh'),
```

Comment:
@brianseeders when I was looking at this I realized we are doing the same thing in both the `jenkins_baseline.sh` and `jenkins_xpack_baseline.sh` so I removed this script